source/write/insert.txt

```diff
@@ -0,0 +1,167 @@
+from your {+language+} application, create a ``Mongo::Client`` object that connects to an Atlas cluster
+and assign the following values to your ``database`` and ``collection`` variables:
+
+.. literalinclude:: /includes/write/insert.rb
+    :language: ruby
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+The _id Field
+-------------
+
+In a MongoDB collection, each document *must* contain an ``_id`` field
+with a unique field value.
+
+MongoDB allows you to manage this field in two ways:
+
+- Set the ``_id`` field for each document yourself, ensuring each
+  value is unique.
+- Let the driver automatically generate unique ``BSON::ObjectId``
+  values for each document ``_id`` field.
+
+Unless you can guarantee uniqueness, we recommend
+letting the driver automatically generate ``_id`` values.
+
+.. note::
+
+   Duplicate ``_id`` values violate unique index constraints, which
+   causes the driver to return an error.
+
+To learn more about the ``_id`` field, see the
+:manual:`Unique Indexes </core/index-unique/>` guide in the {+mdb-server+} manual.
+
+To learn more about document structure and rules, see the
+:manual:`Documents </core/document>` guide in the {+mdb-server+} manual.
+
+Insert One Document
+-------------------
+
+To add a single document to a MongoDB collection, call the ``insert_one``
+method and pass the document you want to insert.
+
+The following example inserts a document into the ``restaurants`` collection:
+
+.. literalinclude:: /includes/write/insert.rb
+   :language: ruby
+   :start-after: start-insert-one
+   :end-before: end-insert-one
+   :dedent:
+   :copyable:
+
+Insert Multiple Documents
+-------------------------
+
+To add multiple documents to a MongoDB collection, call the ``insert_many``
+function and pass a list of documents you want to insert.
+
+The following example inserts two documents into the ``restaurants`` collection:
+
+.. literalinclude:: /includes/write/insert.rb
+   :language: ruby
+   :start-after: start-insert-many
+   :end-before: end-insert-many
+   :dedent:
+   :copyable:
+
+Modify Insert Behavior
+----------------------
+
+You can pass a ``Hash`` object as a parameter to the ``insert_one``
+method to set options to configure the insert operation. If you don't specify any options,
+the driver performs the insert operation with default settings.
+
+The following table describes the options you can set to
+configure the ``insert_one`` operation:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``bypass_document_validation``
+     - | Instructs the driver whether or not to ignore
+         :manual:`document-level validation </core/schema-validation>`. 
+       | Defaults to ``false``.
```

Comment:
S: instead of inline linking to the Server page, you could use the same "For more information ..." syntax as the other table descriptions (applies to write_concern too)
```suggestion
     - | Instructs the driver whether or not to ignore document-level
          validation. For more information, see
         :manual:`Schema Validation </core/schema-validation>`. 
         in the {+mdb-server+} manual.
       | Defaults to ``false``.
```